.github/workflows/release.yml

```diff
@@ -50,6 +50,15 @@ jobs:
           - build: linux-arm
             os: ubuntu-22.04
             target: aarch64-unknown-linux-gnu
+          - build: linux-arm
+            os: ubuntu-22.04
+            target: aarch64-unknown-linux-musl
+          - build: linux-armv7
+            os: ubuntu-22.04
+            target: arm-unknown-linux-gnueabi
```

Comment:
@patrickelectric there seem to be armv7 specific targets (https://doc.rust-lang.org/rustc/platform-support.html), should they be used here instead?